wasm/js/webrtc.js

```diff
@@ -109,6 +181,26 @@
 			}
 		},
 
+		rtcLocalDescriptionSdp: function(pc) {
```

Comment:
Could you please rename `rtcLocalDescriptionSdp` to `rtcAllocLocalDescription`, to both match the semantics of the C API of libdatachannel and indicates it allocates the memory?